azkaban-common/src/test/java/azkaban/user/PermissionTest.java

```diff
@@ -105,10 +105,28 @@ public void testFlags() throws Exception {
     final int flags = permission.toFlags();
     final Permission permission2 = new Permission(flags);
 
-    assertTrue(permission2.isPermissionSet(Type.READ));
-    assertTrue(permission2.isPermissionSet(Type.WRITE));
+    assertThat(permission2.isPermissionSet(Type.READ)).isTrue();
+    assertThat(permission2.isPermissionSet(Type.WRITE)).isTrue();
 
-    assertTrue(permission.equals(permission2));
+    assertThat(permission.equals(permission2)).isTrue();
+  }
+
+  @Test
+  public void testUploadProjectFlag() throws Exception {
```

Comment:
Test ensures that the binary bit for UPLOADPROJECTS permission is not turned on by setting other permissions.